### PR TITLE
Update centreon doc to 18.9

### PR DESCRIPTION
**Pull request by adr-mo**
* Update commands (el6 support dropped)
* Add missing commands (php 7 related)
* Reorganize steps of the installation process
* Update installation documentation via web interface

### CODE REVIEW

File: doc/en/installation/from_packages.rst [hunk 1]
```diff
@@ -21,153 +21,144 @@ SELinux should be disabled; for this, you have to modify the file */etc/selinux/
...
+::
+
+    $ getenforce
+    Disabled
+
+
 ******************
-Repository install
+Repository installation
 ******************
```
Comment (lpinsivy) [inline doc/en/installation/from_packages.rst]: For title tags you need to put '*' until the last charectère

File: doc/fr/installation/from_packages.rst [hunk 2]
```diff
@@ -54,117 +75,82 @@ Exécutez la commande :
...
+::
 
-  $ ssh-copy-id centreon@your_poller_ip
+   $ yum install MariaDB-server
+   $ systemctl restart mysql
 
-Installer MySQL sur le même serveur
------------------------------------
+Système de gestion de base de données
+-------------------------------------
```
Comment (lpinsivy) [inline doc/fr/installation/from_packages.rst]: not enough '-'

File: doc/fr/installation/from_packages.rst [hunk 3]
```diff
@@ -23,24 +23,45 @@ SELinux doit être désactivé. Pour cela vous devez modifier le fichier */etc/s
...
+droits suffisants.
+
+::
 
+   $ yum install centos-release-scl
 
 Le dépôt est maintenant installé.
 
+Dépôt Centreon
+---------------------------------------
```
Comment (lpinsivy) [inline doc/fr/installation/from_packages.rst]: Too much '-'

File: doc/fr/installation/from_packages.rst [hunk 4]
```diff
@@ -23,24 +23,45 @@ SELinux doit être désactivé. Pour cela vous devez modifier le fichier */etc/s
...
+
 ***********************
 Installation des dépôts
 ***********************
 
-Afin d'installer les logiciels Centreon à partir des dépôts, vous devez au préalable installer
-le fichier lié au dépôt. Exécutez la commande suivante à partir d'un utilisateur possédant les
-droits suffisants.
+Dépôt *Software collections* de Red Hat
+---------------------------------------
```
Comment (lpinsivy) [inline doc/fr/installation/from_packages.rst]: two '-' must be deleted

File: doc/en/installation/from_packages.rst [hunk 5]
```diff
@@ -21,153 +21,144 @@ SELinux should be disabled; for this, you have to modify the file */etc/selinux/
...
+   # mkdir -p  /etc/systemd/system/mariadb.service.d/
+   # echo -ne "[Service]\nLimitNOFILE=32000\n" | tee /etc/systemd/system/mariadb.service.d/limits.conf
+   # systemctl daemon-reload
+   # service restart mysql
 
 PHP timezone
 ------------
 
-PHP timezone should be set; go to /etc/php.d directory and create a file named php-timezone.ini which contains the following line :
+PHP timezone needs to be set; to do so go to /etc/opt/rh/rh-php71/php.d directory and create a file named php-timezone.ini which contains the following line :
```
Comment (sc979) [inline doc/en/installation/from_packages.rst]: which contains the following line : -> which contains the timezone of the centreon's server. For example :

File: doc/fr/installation/from_packages.rst [hunk 6]
```diff
@@ -23,24 +23,45 @@ SELinux doit être désactivé. Pour cela vous devez modifier le fichier */etc/s
...
-Pour CentOS 7.
+Afin d'installer les logiciels Centreon, le dépôt *Software collections* de Red Hat doit être activé.
 
-::
+.. note::
+    Le dépôt *Software collections* est nécessaire pour l'installation de php 7 et les librairies associées.
 
-   $ wget http://yum.centreon.com/standard/3.4/el7/stable/noarch/RPMS/centreon-release-3.4-4.el7.centos.noarch.rpm
-   $ yum install --nogpgcheck centreon-release-3.4-4.el7.centos.noarch.rpm
+Exécutez la commande suivante à partir d'un utilisateur possédant les
```
Comment (sc979) [inline doc/fr/installation/from_packages.rst]: possédant les ... -> disposant des ...

File: doc/fr/installation/from_packages.rst [hunk 7]
```diff
@@ -23,24 +23,45 @@ SELinux doit être désactivé. Pour cela vous devez modifier le fichier */etc/s
...
 
+   $ yum install centos-release-scl
 
 Le dépôt est maintenant installé.
 
+Dépôt Centreon
+---------------------------------------
+
+Afin d'installer les logiciels Centreon à partir des dépôts, vous devez au préalable installer
+le fichier lié au dépôt. Exécutez la commande suivante à partir d'un utilisateur possédant les
```
Comment (sc979) [inline doc/fr/installation/from_packages.rst]: possédant les ... -> disposant des ...

File: doc/en/installation/from_packages.rst (hunk 1 above)
Comment (adr-mo) [reply doc/en/installation/from_packages.rst]: done

File: doc/fr/installation/from_packages.rst (hunk 2 above)
Comment (adr-mo) [reply doc/fr/installation/from_packages.rst]: done

File: doc/fr/installation/from_packages.rst (hunk 3 above)
Comment (adr-mo) [reply doc/fr/installation/from_packages.rst]: done

File: doc/fr/installation/from_packages.rst (hunk 4 above)
Comment (adr-mo) [reply doc/fr/installation/from_packages.rst]: done

File: doc/en/installation/from_packages.rst (hunk 5 above)
Comment (adr-mo) [reply doc/en/installation/from_packages.rst]: done

File: doc/fr/installation/from_packages.rst (hunk 7 above)
Comment (adr-mo) [reply doc/fr/installation/from_packages.rst]: done

File: doc/fr/installation/from_packages.rst (hunk 6 above)
Comment (adr-mo) [reply doc/fr/installation/from_packages.rst]: done

File: doc/en/installation/from_packages.rst [hunk 8]
```diff
@@ -96,78 +93,71 @@ Perform the command:
...
+Setting this option into /etc/my.cnf will NOT work.
 
+Perform instead:
+
+::
+
+   # mkdir -p  /etc/systemd/system/mariadb.service.d/
+   # echo -ne "[Service]\nLimitNOFILE=32000\n" | tee /etc/systemd/system/mariadb.service.d/limits.conf
+   # systemctl daemon-reload
+   # service restart mysql
```
Comment (lgcosta) [inline doc/en/installation/from_packages.rst]: Use standard `systemctl` as how is the whole document.

File: doc/en/installation/from_packages.rst [hunk 9]
```diff
@@ -96,78 +93,71 @@ Perform the command:
...
-::
-
-   # mkdir -p  /etc/systemd/system/mariadb.service.d/
-   # echo -ne "[Service]\nLimitNOFILE=32000\n" | tee /etc/systemd/system/mariadb.service.d/limits.conf
-   # systemctl daemon-reload
-   # service mysql restart
-
-Launch services during the system startup
+Launch services during the system bootup
 -----------------------------------------
```
Comment (lgcosta) [inline doc/en/installation/from_packages.rst]: remove a one '-'

Comment (adr-mo) [reply doc/en/installation/from_packages.rst]: done thanks !

File: doc/en/installation/from_packages.rst (hunk 8 above)
Comment (adr-mo) [reply doc/en/installation/from_packages.rst]: done thanks !

**Pull request merged**